src/Entity/Message.php

```diff
@@ -409,4 +410,16 @@ public function setLanguage($language) {
     $this->language = $language;
   }
 
+  /**
+   * Default value callback for 'uid' base field definition.
+   *
+   * @see ::baseFieldDefinitions()
```

Comment:
If you see the `getCurrentUserId()` in `Drupal\node\Entity\Node` class. I think it's might be best to remove or in place of this comment, state it's use in other modules? Is it ok to add even though Message does not use it directly, *yet*? I thought it might be best place for it, by using core for inspiration.